retrofit/src/main/java/retrofit/client/Request.java

```diff
@@ -56,8 +51,8 @@ public String getUrl() {
     return url;
   }
 
-  /** Returns an unmodifiable list of headers, never {@code null}. */
-  public List<Header> getHeaders() {
+  /** Returns an unmodifiable list of headers. May be {@code null}. */
```

Comment:
Yep it's never null. Will fix.
On Jan 30, 2015 6:10 PM, "Jesse Wilson" <notifications@github.com> wrote:

> In retrofit/src/main/java/retrofit/client/Request.java
> <https://github.com/square/retrofit/pull/729#discussion_r23885352>:
>
> > @@ -56,8 +51,8 @@ public String getUrl() {
> >      return url;
> >    }
> >
> > -  /** Returns an unmodifiable list of headers, never {@code null}. */
> > -  public List<Header> getHeaders() {
> > +  /** Returns an unmodifiable list of headers. May be {@code null}. */
>
> nits: its *always* unmodifiable, and it's not a list
>
> non-nit: return empty headers instead of null?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/retrofit/pull/729/files#r23885352>.
>